cfgov/v1/models/sublanding_page.py

```diff
@@ -75,10 +75,8 @@ def get_browsefilterable_posts(self, request, limit):
                 filtered_controls.update({id: []})
             form_class = page.get_form_class()
             posts = page.get_page_set(form_class(parent=page, hostname=request.site.hostname), request.site.hostname)
-            if filtered_controls[id]:
-                filtered_controls[id] += posts
-            else:
-                filtered_controls[id] = posts
-        posts_tuple_list = [(id, post) for id, posts in filtered_controls.iteritems() for post in posts]
+            filtered_controls[id].append(posts)
+
+        posts_tuple_list = [(id, post) for id, posts in filtered_controls.iteritems() for post in chain(*posts)]
```

Comment:
Sure. I made the smallest change consistent with the behavior as I understood it, but I can certainly pretty it up.